docs/sources/get-started/ride-share-tutorial.md

```diff
@@ -0,0 +1,274 @@
+{{< /admonition >}}
+{{</docs/ignore>}}
+
+<!-- INTERACTIVE page intro.md END -->
+
+<!-- INTERACTIVE page step1.md START -->
+
+## Clone the repository
+
+1. Clone the repository to your local machine:
+
+    ```bash
+    git clone https://github.com/grafana/pyroscope.git && cd pyroscope
+    ```
+
+1. Navigate to the tutorial directory:
+
+    ```bash
+    cd examples/language-sdk-instrumentation/python/rideshare/flask
+    ```
+
+## Start the application
+
+Start the application using Docker Compose:
+
+```bash
+docker compose up -d
+```
+
+This may take a few minutes to download the required images and build the demo application. Once ready, you will see the following output:
+
+```console
+ ✔ Network flask_default  Created
+ ✔ Container flask-ap-south-1  Started
+ ✔ Container flask-grafana-1  Started
+ ✔ Container flask-pyroscope-1  Started     
+ ✔ Container flask-load-generator-1 Started 
+ ✔ Container flask-eu-north-1 Started       
+ ✔ Container flask-us-east-1 Started  
+```
+
+Optional: To verify the containers are running, run:
+
+```bash
+docker ps -a
+```
+<!-- INTERACTIVE page step1.md END -->
+
+<!-- INTERACTIVE page step2.md START -->
+
+## Accessing the Profile Explorer in Grafana
+
+Grafana includes a Profiles Explorer that you can use to view the profile data. To access the Profiles Explorer, open a browser and navigate to [http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer](http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer).
+
+### How tagging works
+
+In this example, the application is instrumented with Pyroscope using the Python SDK.
+The SDK allows you to tag functions with metadata that can be used to filter and group the profile data in the Pyroscope UI.
+This example uses static and dynamic tagging.
+
+To start, let's take a look at a static tag use case. Within the `server.py` file, find the Pyroscope configuration:
+
+```python
+ pyroscope.configure(
+     application_name = app_name,
+     server_address   = server_addr,
+     basic_auth_username = basic_auth_username, # for grafana cloud
+     basic_auth_password = basic_auth_password, # for grafana cloud
+     tags             = {
+         "region":   f'{os.getenv("REGION")}',
+     }
+ )
+```
+This tag is considered static is because the tag is set at the start of the application and doesn't change.
+In this case, it's useful for grouping profiles on a per region basis, which lets yousee the performance of the application per region.
+
+Open Grafana to see how this tag is used:
+
+1. Open the Profile Explorer in Grafana UI, which can be accessed using the following url: [http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer](http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer).
+2. Select on `Labels` in the `Exploration` path.
+3. Select the `region` tab in the `Group by labels` section. 
```

Comment:
Adds an extra step to explain how to get to Explore Profiles in the UI. We also bold any UI text. For more information, refer to https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold

```suggestion
1. Open Grafana using the following url: [http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer](http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer).
1. In the main menu, select **Explore** > **Profiles**. 
1. Select  **Labels** in the **Exploration** path.
1. Select the **region** tab in the **Group by labels** section. 
```